frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.jsx

```diff
@@ -126,23 +127,23 @@ export default class TagEditorParam extends Component {
       isDimension && Array.isArray(tag.dimension);
     const hasWidgetOptions = widgetOptions && widgetOptions.length > 0;
     return (
-      <div className="pb2 mb2 border-bottom border-dark">
-        <h3 className="pb2">{tag.name}</h3>
+      <Card className="p2 mb2">
+        <h4 className="pb2 text-monospace text-brand">{tag.name}</h4>
```

Comment:
I'm not a huge fan of the use of monospace here. I'd just stick with a normal header, maybe even bump it up to a h3. 
```suggestion
        <h4 className="pb2 text-brand">{tag.name}</h4>
```

Reply:
Sounds good.